specification/transport_layer/transport_layer.tex

```diff
@@ -575,6 +602,81 @@ \subsection{Transfer-ID comparison}\label{sec:transfer_id_forward_distance}
 \end{minted}
 \end{minipage}
 
+\subsection{Payload truncation}
+
+The deterministic nature of UAVCAN in general and DSDL in particular allows implementations to statically determine the
+maximum amount of memory that is required to contain a data object of a particular type.
+Consequently, an implementation that is interested in receiving data objects of a particular type\footnote{%
+    Messages, service requests, or service responses.
+}
+can statically determine the maximum length of the transfer payload.
+
+Implementations shall handle incoming transfers containing a larger amount of payload data than expected.
+In the event of such extra payload being received, a compliant implementation should
+discard the excessive (unexpected) data at the end of the received payload\footnote{%
```

Comment:
should we clarify that, for multi-frame transfers, the excessive data must be used to calculate the CRC before discarding it?

Reply:
Do I understand correctly that you want to replace or amend the following?

> The transfer CRC, if applicable, shall be validated regardless of the presence of the extra payload in the transfer.